scripts/validate-manifest.sh

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env bash
+#
+# validate-manifest.sh [path/to/manifest.json]
+#
+# Dependencies: jq tr sort uniq
+#
+# Description: Ensures consistency of versions specified in manifest.json.
+
+set -eou pipefail
+
+if ! command -v jq > /dev/null; then
+    echo "This script requires jq to be available."
+    exit 1
+fi
+
+GIT_ROOT=$(git rev-parse --show-toplevel)
+MANIFEST_JSON_PATH=${1:-"${GIT_ROOT}/manifest.json"}
+
+declare -a MANIFEST_VERSIONS
+declare -a ROUTE_VERSIONS
+
+# Read the versions array in manifest.json
+readarray -t MANIFEST_VERSIONS < <(jq -r '
```

Comment:
I believe `readarray` sets a bash version requirement, but not sure when it was added.

Reply:
Seems to have been added in 4.x; added a preflight check for Bash version.